packages/python/plotly/plotly/tests/test_core/test_graph_objs/test_performance.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import time
+from unittest import TestCase
+import pytest
+import numpy as np
+import plotly.graph_objs as go
+import plotly.io as pio
+from plotly.tests.b64 import b64, _b64
+
+
+def test_performance_b64():
+    rand_arr_1 = np.random.random(100000)
+    rand_arr_2 = np.random.random(100000)
+    raw_arr_1 = rand_arr_1.tolist()
```

Comment:
Using `raw` for this variable is rather confusing; whereas raw data here should point to raw user data in numpy format.
Therefore I'd suggest we use `normal_arr` instead of `raw_arr`.

Reply:
I changed it to `list_1` and `list_2`, does that work?

Reply:
Perfect. Less is more.